pyperformance/data-files/benchmarks/bm_richards_super/run_benchmark.py

```diff
@@ -0,0 +1,430 @@
+"""
+based on a Java version:
+ Based on original version written in BCPL by Dr Martin Richards
+ in 1981 at Cambridge University Computer Laboratory, England
+ and a C++ version derived from a Smalltalk version written by
+ L Peter Deutsch.
+ Java version:  Copyright (C) 1995 Sun Microsystems, Inc.
+ Translation from C++, Mario Wolczko
+ Outer loop added by Alex Jacoby
+ super() usage added by Carl Meyer
+"""
+
+import pyperf
+
+
+# Task IDs
+I_IDLE = 1
+I_WORK = 2
+I_HANDLERA = 3
+I_HANDLERB = 4
+I_DEVA = 5
+I_DEVB = 6
+
+# Packet types
+K_DEV = 1000
+K_WORK = 1001
+
+# Packet
+
+BUFSIZE = 4
+
+BUFSIZE_RANGE = range(BUFSIZE)
+
+
+class Packet(object):
```

Comment:
Since this is a new benchmark, and we won't be comparing the results of this benchmark to the results of the old benchmark, I would remove explicit inheritance from `object` in benchmark, so that it looks like idiomatic, modern Python code:

```suggestion
class Packet:
```

(etc. throughout the PR)